app/backend/hiStoryBackend/api/views/auth/views_v1.py

```diff
@@ -24,6 +24,9 @@ def sign_in(request):
 	else:
 		return jrh.bad_request(["username and email are both missing"])
 
+	if 'password' not in content:
```

Comment:
It would be better to check this before a database query happens. If this key is missing, there is no need to retrieve the user from the database.